igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/gitlab/client/model/Commit.groovy

```diff
@@ -0,0 +1,19 @@
+package com.netflix.spinnaker.igor.scm.gitlab.client.model
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
```

Comment:
I would strongly consider to use Java 8 Time & Date API (JSR 310) instead of Joda Time. I _think_ Jackson converters are already included on the class path. If not, just include `compile spinnaker.dependency('jacksonJsr310')` in the gradle config.

Reply:
JodaTime is also used in the code that talks to BitBucket. In the interest of consistency we decided to use it here as well. Perhaps we could do a separate PR to rid the project of JodaTime al together?

Reply:
I'm fine with that 👍